src/general_ode_solve.jl

```diff
@@ -0,0 +1,112 @@
+struct NNGenODE{C,IC,O,K} <: NeuralNetDiffEqAlgorithm

```

Comment:
why does this need to be different from the other one?

Reply:
yeah, it is the hard code. It also will be deleted.